src/components/Footer/Footer.js

```diff
@@ -1,15 +1,82 @@
+import Link from 'next/link';
+
+import useSite from 'hooks/use-site';
+import { postPathBySlug } from 'lib/posts';
+import { categoryPathBySlug } from 'lib/categories';
+
+import Section from 'components/Section';
+import Container from 'components/Container';
+
 import styles from './Footer.module.scss';
 
 const Footer = () => {
+  const { metadata = {}, recentPosts = [], categories = [] } = useSite();
+  const { title } = metadata;
+
+  const hasRecentPosts = Array.isArray(recentPosts) && recentPosts.length > 0;
+  const hasRecentCategories = Array.isArray(categories) && categories.length > 0;
+  const hasMenu = hasRecentPosts || hasRecentCategories;
+
   return (
     <footer className={styles.footer}>
-      <a
-        href="https://vercel.com?utm_source=create-next-app&utm_medium=next-wordpress-starter&utm_campaign=create-next-app"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        Powered by <img src="/vercel.svg" alt="Vercel Logo" />
-      </a>
+      {hasMenu && (
+        <Section className={styles.footerMenu}>
+          <Container>
+            <ul className={styles.footerMenuColumns}>
+              {hasRecentPosts && (
+                <li>
+                  <Link href="/posts/">
```

Comment:
Really like this footer - I wonder if it's worth opting in though? 

It's looking great but getting opinionated. :)

Reply:
after chatting in discord it seems like we're good with this since it can easily be removed if people dont like it and its pretty common with WP projects